frontend/src/components/shared/ProgressBar/ProgressBar.tsx

```diff
@@ -80,7 +82,10 @@ function ProgressBar({
     },
     BarProgress: {
       style: ({ $theme }: { $theme: any }) => ({
-        backgroundColor: theme.colors.blue70,
+        backgroundColor:
+          activeTheme.name === "Custom Theme"
+            ? theme.colors.primary
+            : theme.colors.blue70,
```

Comment:
Same comment as above

Reply:
Updated in [b87c19b](https://github.com/streamlit/streamlit/pull/5086/commits/b87c19b15edad1ab2db56d59bd9ae0f24d5f136e)